constants/theme.js

```diff
@@ -26,4 +26,8 @@ export const THEME = {
   },
 }
 
-export const FontSize14InRem = '0.875rem'
+export const fontSize = {
+  12: '0.75rem',
+  14: '0.875rem',
+  16: '1rem',
+}
```

Comment:
```suggestion
export const fontSize = {
  12: '0.75rem',
  14: '0.875rem',
}
```
MUI already has 1rem set as their default

Reply:
@mikestone14 this hasn't been resolved yet

Reply:
yeah but we might want to specify a font size for a MUI component that doesn't use that default.

Reply:
we should be able to use "typography.fontSize" for the default

Reply:
ok, it has been removed.